### PR TITLE
Add more community example

### PR DESCRIPTION
**Pull request by shirshak55**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ A curated list of examples related to actix.
 * [Roseline](https://github.com/DoumanAsh/roseline.rs) : A personal web site and discord & IRC bot to access simple SQLite database. Demonstrates usage of various actix and actix-web concepts.
 * [Actix Auth Server](https://hgill.io/posts/auth-microservice-rust-actix-web-diesel-complete-tutorial-part-1/) : Auth web micro-service with rust using actix-web - complete tutorial. See code in [examples/simple-auth-server](https://github.com/actix/examples/tree/master/simple-auth-server)
 * [actix-raft](https://github.com/railgun-rs/actix-raft) : An implementation of the Raft consensus protocol using the actix Actor framework.
+* [lemmy](https://github.com/dessalines/lemmy) : uilding a federated alternative to reddit in rust.
```
Comment (vmalloc) [inline README.md]: Should be "Building", and probably just "A federated alternative.."

File: README.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ A curated list of examples related to actix.
...
+* [actix-realworld-example-app](https://github.com/fairingrey/actix-realworld-example-app) : Implementation of the RealWorld backend API spec in Actix, Rust's powerful actor system and most fun web framework.
```
Comment (vmalloc) [inline README.md]: I'd lose the "Rust's powerful..." suffix here

File: README.md (hunk 1 above)
Comment (shirshak55) [reply README.md]: yep :)

**Comment by JohnTitor** [comment]
Thanks!

**Pull request merged**